### DISCUSSION TITLE
Research.

### DISCUSSION THREAD

**Discussion by gitwithsuraj** [gitwithsuraj/MINOR.1_QuickLog#2] [General]
I did some research guys and found a link that explains what is log analysis 
https://www.vmware.com/topics/log-analysis

**Comment by Bhavyajain0604** [comment]
Guys I found some research papers which could be relevant to our topic and can help us in literature survey

**Comment by Bhavyajain0604** [comment]
[logmine.pdf](https://github.com/user-attachments/files/16870355/logmine.pdf)

**Comment by Bhavyajain0604** [comment]
[quick,_efficient_and_effective_patient_specific.24.pdf](https://github.com/user-attachments/files/16870359/quick._efficient_and_effective_patient_specific.24.pdf)

**Comment by Bhavyajain0604** [comment]
[2311.11809v2.pdf](https://github.com/user-attachments/files/16870367/2311.11809v2.pdf)

**Comment by Bhavyajain0604** [comment]
[J Applied Clin Med Phys - 2002 - Litzenberg - Verification of dynamic and segmental IMRT delivery by dynamic log file.pdf](https://github.com/user-attachments/files/16870370/J.Applied.Clin.Med.Phys.-.2002.-.Litzenberg.-.Verification.of.dynamic.and.segmental.IMRT.delivery.by.dynamic.log.file.pdf)

**Comment by gitwithsuraj** [comment]
thank you bhavya, i have read these reserch paper which you have posted and i think they are helpful in the resech of our minor project

**Comment by Bhavyajain0604** [comment]
Guys I found something which could  be really helpfull to us in our literaturesurver and for tool and technology when you will get time so please look into this

**Comment by Bhavyajain0604** [comment]
[jansen_search_log_analysis.pdf](https://github.com/user-attachments/files/16870932/jansen_search_log_analysis.pdf)

**Comment by gitwithsuraj** [comment]
http://regex101.com/r/cA4wE0/10
